scripts/functions.js

```diff
@@ -0,0 +1,156 @@
+//Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+const tvBrands = inventory.map((tv) => {
+    return tv.brand;
+})
+console.log(tvBrands)
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+const tvSoldOut = inventory.filter((arr) => {
+    return arr.originalStock - arr.sold === 0;
+})
```

Comment:
Je zou dit nog wat korter kunnen schrijven
```suggestion
const tvSoldOut = inventory.filter(arr => arr.originalStock === arr.sold);
```